assets/src/js/lightbox.js

```diff
@@ -0,0 +1,108 @@
+const GRID_GALLERY_CLASS = 'gallery-grid';
+const SLIDER_GALLERY_CLASS = 'carousel-wrap';
+const THREE_COLUMN_GALLERY_CLASS = 'three-column-images';
+
+const setupPhotoSwipe = function(items, clickedIndex) {
+  var pswpElement = $('.pswp')[0];
```

Comment:
I know it's not really a big deal, but wouldn't it be nicer to use `const/let` in this function instead of `var`? It would also be more consistent with the rest of the file :)

Reply:
Revisiting this old PR, indeed, I was just copying their example on the site. 👍